releasenotes/notes/dag-oxide-60b3d7219cb21703.yaml

```
@@ -0,0 +1,49 @@
+---
+features_transpiler:
+  - |
+    The implementation of the :class:`.DAGCircuit` has been rewritten in Rust. This rewrite of
+    the Python class should be fully API compatible with the previous Python implementation of
+    the class. While the class was previously implemented using
+    `rustworkx <https://www.rustworkx.org/>`__ and it's underlying data graph structure existed
+    in Rust, the implementation of the class and all the data was stored in Python. This new
+    version of :class:`.DAGCircuit` stores a Rust native representation for all it's data and
+    is fully implemented in Rust. This new implementation should be more efficient in memory
```

```suggestion
    is fully implemented in Rust. This should be more memory efficient
```
Thinking like an editor, looking for fat to trim.

---

I didn't apply this because the sentence doesn't flow with L11 anymore. I'm fine trimming it like this but we'll need to update the start of L11 to remove "usage" too.